API-GUIDELINE.md

```diff
@@ -0,0 +1,254 @@
+#API GUIDELINES
+
+##Overview
+As commercetools aims to be 100% API First, developer experience for API solution becomes very important for us.
```

Comment:
> developer experience for API solution
I don't really understand this sentence. What do you mean by that?

Reply:
I would rephrase saying: "As commercetools aims to be 100% API First, it's important to have a well designed developer experience"

Reply:
"As commercetools is 100% API first, it's important to have a well designed developer experience"
😉

Reply:
🥇